common/app/views/support/GetClasses.scala

```diff
@@ -54,13 +60,42 @@ object GetClasses {
         ("fc-item--is-commentable", item.discussionSettings.isCommentable),
         ("fc-item--is-media-link", item.isMediaLink),
         ("fc-item--has-video-main-media", item.hasVideoMainMedia),
-        ("fc-item--dynamic-layout", isDynamic && item.cardTypes.canBeDynamicLayout && !item.cutOut.isDefined),
+        ("fc-item--is-dynamic-card", isDynamic && item.cardTypes.canBeDynamicLayout && !item.cutOut.isDefined),
+        ("fc-item--has-floating-sublinks", hasFloatingSublinks(item, isDynamic)),
       ) ++ item.snapStuff.map(_.cssClasses.map(_ -> true).toMap).getOrElse(Map.empty)
         ++ mediaTypeClass(item).map(_ -> true)
         ++ adFeatureMediaClass(item).map(_ -> true),
     )
   }
 
+  def hasFloatingSublinks(item: ContentCard, isDynamic: Boolean) = {
```

Comment:
I still think this method could live in `ContentCard` and just take `isDynamic` as an argument if necessary.